security-alert-notifier.rb

```diff
@@ -177,17 +181,33 @@ def fetch_repositories(http:, cursor: nil)
     github = GitHub.new
 
     if github.vulnerable_repos.any?
-      total_vulnerabilities = github.vulnerable_repos.sum { |repo| repo.alerts.length }
-      puts "WARNING: #{total_vulnerabilities} vulnerabilities in #{github.vulnerable_repos.length} repos"
+      if options[:repos].nil?
+        total_vulnerabilities = github.vulnerable_repos.sum { |repo| repo.alerts.length }
+        vulnerable_repo_count = github.vulnerable_repos.length
+      else
+        total_vulnerabilities = 0
+        vulnerable_repos = 0
+        github.vulnerable_repos.each do |repo|
+          if options[:repos].nil? || repo.url =~ /#{options[:repos]}/
+            vulnerable_repos = vulnerable_repos + 1
+            repo.alerts.each do |alert|
+              total_vulnerabilities = total_vulnerabilities + 1
```

Comment:
```suggestion
              vulnerable_repo_count = vulnerable_repo_count + 1
```
rename for better voriable name